data/mods/gen9ssb/random-teams.ts

```diff
@@ -1009,10 +1016,17 @@ export class RandomStaffBrosTeams extends RandomTeams {
 					this.sample(this.dex.types.names()) :
 					this.sampleIfArray(ssbSet.teraType);
 			}
-			const moves = [];
+			const moves: string[] = [];
 			while (moves.length < 3 && ssbSet.moves.length > 0) {
 				let move = this.sampleNoReplace(ssbSet.moves);
 				if (Array.isArray(move)) move = this.sampleNoReplace(move);
+				if (ssbSet.species === "Vespiquen") {
+					if (move === 'Calm Mind' && moves.includes('Night Shade')) {
+						move = this.sample(['Thief', 'Toxic']);
+					} else if (move === 'Night Shade' && moves.includes('Calm Mind')) {
+						move = 'Bug Buzz';
+					}
+				}
```

Comment:
```suggestion
				if (ssbSet.species === "Vespiquen") {
					if (move === 'Calm Mind' && moves.includes('Night Shade')) {
						move = this.sample(['Thief', 'Toxic']);
					} else if (move === 'Night Shade' && moves.includes('Calm Mind')) {
						move = 'Bug Buzz';
					}
				}
```
There is a dedicated section for set specific tweaks down further, migrate the code there